### PR TITLE
Sass refactor

### PR DESCRIPTION
**Pull request by CalebCyphers**
## Description

This push will add the html for the pantry page, as well as begin the refactoring of css into scss. The scss files are saved within the /css folder.

## Affected areas of application

- pantry.html
- index.html
- styles.scss
- styles.css

## How Has This Been Tested?

- I have been testing in chrome. The difference between the regular view and the mobile view in the dev tools has been quite significant, and is something I'm working on currently.

## Relevant Tickets

- no tickets are closed by this pr

## Screenshots

main page after partial sass refactor
Screen Shot 2020-09-12 at 2 15 12 PM

### CODE REVIEW

File: src/css/styles.scss [hunk 1]
```diff
@@ -1,100 +1,198 @@
...
+  align-items: center;
+}
+
 html {
-  background-color: #37515C;
-  border: 2px solid #000;
-  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  background-color: $secondary-color;
```
Comment (taylorjohnson141) [inline src/css/styles.scss:16]: I like the use of Sass variables

File: src/index.html [hunk 2]
```diff
@@ -8,42 +8,45 @@
     <link rel="stylesheet" href="./styles.css">
   </head>
   <body>
-
+    <section class="user-info">
+      <h1>Hi, I'm John</h1>
```
Comment (nicolegooden) [inline src/index.html:12]: we should get rid of this temporary text and leave it empty for the DOM display of a random user's name

File: src/css/styles.scss [hunk 3]
```diff
@@ -1,100 +1,198 @@
+$primary-color: #cff2da;
+$secondary-color: #37515C;
+$tertiary-color: #C3C2C6;
+$highlight-color: #EBF5EE;
+$darkest-color: #000;
+
+@mixin centerFlex {
```
Comment (taylorjohnson141) [inline src/css/styles.scss:7]: Nice Job with this

**Review by taylorjohnson141 — APPROVED** [review]

File: src/index.html [hunk 4]
```diff
@@ -8,42 +8,45 @@
...
+      </section>
     <header class='main-header'>
       <div class="wrapper">
         <img class="hamburger-menu" src="./images/hamburger-icon.png" alt="hamburger icon serving as menu button">
-        <img class="x-menu hidden" src="./images/x-icon.png" alt="x icon to be clicked when you are done viewing the menu options">
```
Comment (nicolegooden) [inline src/index.html:15]: We should probably plan to have these `.hidden` classes on our elements by default, then remove them on a toggle when the hamburger icon is clicked.

**Review by nicolegooden — APPROVED** [review]

**Pull request merged**